Bottom: 3ae05c77d04184b91e2fb1ba350eb81038d16615
Top:    da0c5ff5c4e3c9e4460baebe8b142e53bb181119
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-16 18:22:33 +0800

Refresh of ptr_ring-introduce-batch

---

diff --git a/include/linux/ptr_ring.h b/include/linux/ptr_ring.h
index 4211c85..47b0843 100644
--- a/include/linux/ptr_ring.h
+++ b/include/linux/ptr_ring.h
@@ -337,8 +337,8 @@ static inline int ptr_ring_consume_batched_irq(struct ptr_ring *r,
 	return ret;
 }
 
-static inline int ptr_ring_consume_irq_any(struct ptr_ring *r,
-                                           void **array, int n)
+static inline int ptr_ring_consume_batched_any(struct ptr_ring *r,
+					       void **array, int n)
 {
 	unsigned long flags;
 	int ret;
